README.md

```diff
@@ -1,11 +1,154 @@
+
+The generated nuget file(s) will be located in the `bin\Debug` subfolder:
+
+`D:\corefx-testdata\System.IO.Compression.TestData\bin\Debug\System.IO.Compression.TestData.1.0.10.nupkg`
+
+
+#### 3. Test that corefx can import the new nuget package file.
+
+We need to make sure our change works by importing the generated nuget package file into our corefx project.
+
+Open the `corefx\external\test-runtime\XUnit.Runtime.depproj` file, find the `<PackageReference>` element for your TestData project and change the version to the same number you used in step 1.
+
+```xml
+<!-- Test Data -->
+<PackageReference Include="System.IO.Compression.TestData" Version="1.0.10" />
+```
+
+Now from the root of the `corefx` project, run the commands that will import the package file:
+
+```
+PS D:\corefx> .\.dotnet\dotnet.exe restore .\external\test-runtime\XUnit.Runtime.depproj -s D:\corefx-testdata\System.IO.Compression.TestData\bin\Debug\
+    Restore completed in 1.08 sec for D:\corefx\external\test-runtime\XUnit.Runtime.depproj.
+
+PS D:\corefx> .\.dotnet\dotnet.exe msbuild .\external\test-runtime\XUnit.Runtime.depproj /t:rebuild
+
+    Microsoft (R) Build Engine version 16.1.67-preview+g13843078ee for .NET Core
+    Copyright (C) Microsoft Corporation. All rights reserved.
+
+    Restore completed in 292.24 ms for D:\corefx\external\test-runtime\XUnit.Runtime.depproj.
+
+     C:\Users\username\.nuget\packages\...
+     C:\Users\username\.nuget\packages\...
+     ...
+```
+
+ Now check if the nuget package was exported correctly. Go to your current user's nuget packages folder, usually located in `C:\Users\yourusername\.nuget\packages`. Find the folder for your TestData and inside you will find the original dependencies plus the new ones you added:
+
+```
+PS D:\corefx> dir C:\Users\yourusername\.nuget\packages\System.IO.Compression.TestData\1.0.10\
+
+    Directory: C:\Users\yourusername\.nuget\packages\System.IO.Compression.TestData\1.0.10
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+d-----        3/25/2019   3:55 PM                content
+-a----        3/25/2019   3:55 PM            130 .nupkg.metadata
+-a----        3/25/2019   3:55 PM       54372842 system.io.compression.testdata.1.0.10.nupkg
+-a----        3/25/2019   3:55 PM             88 system.io.compression.testdata.1.0.10.nupkg.sha512
+-a----        3/25/2019   3:55 PM           1034 system.io.compression.testdata.nuspec
+
+
+PS D:\corefx> dir C:\Users\yourusername\.nuget\packages\System.IO.Compression.TestData\1.0.10\content\GZipTestData\
+
+    Directory: C:\users\yourusername\.nuget\packages\System.IO.Compression.TestData\1.0.10\content\GZipTestData
+
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----        3/25/2019  3:55 PM           1748 example.gz
+
```

Comment:
Looks like you missed a ``` here before the ####4

Reply:
I'll add it separately, I don't know how to comment those characters as code suggestions.